benchmark/bench_RBKI/RBKI_speed_comparisons.cc

```diff
@@ -208,16 +223,16 @@ static void call_all_algs(
 
     // Additional params setup.
     all_algs.RSVD.block_sz = b_sz;
-    all_algs.RBKI.num_threads_some = 4;
-    all_algs.RBKI.num_threads_rest = 48;
     // Matrices R or S that give us the singular value spectrum returned by RBKI will be of size b_sz * num_krylov_iters / 2.
     // These matrices will be full-rank.
     // Hence, target_rank = b_sz * num_krylov_iters / 2 
     // RBKI.max_krylov_iters = (int) ((target_rank * 2) / b_sz);
     // 
     // Instead of the above approach, we now pre-specify the maximum number of Krylov iters that we allow for in num_matmuls.
     all_algs.RBKI.max_krylov_iters = (int) num_matmuls;
-
+    all_algs.RBKI.num_threads_min = 4;
+    all_algs.RBKI.num_threads_max = RandLAPACK::util::get_omp_threads<T>();
```

Comment:
Find-and-replace ``get_omp_threads<T>()`` with ``get_omp_threads()``

Reply:
Fixed.